pandas/core/groupby/groupby.py

```diff
@@ -2675,6 +2674,47 @@ def rank(
         Returns
         -------
         DataFrame with ranking of values within each group
+
+        See Also
+        --------
+        Series.groupby : Apply a function groupby to a Series.
+        DataFrame.groupby : Apply a function groupby
+            to each row or column of a DataFrame.
+        Series.rank : Apply a function rank to a Series.
+        DataFrame.rank : Apply a function rank
+            to each row or column of a DataFrame.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'group': ['a', 'a', 'a', 'a',
+        ...                              'a', 'b', 'b', 'b', 'b', 'b'],
+        ...                    'value': [2, 4, 2, 3, 5, 1, 2, 4, 1, 5]})
```

Comment:
The formatting of this looks a bit awkward, can you use `black` on this (or at least would be clearer if a's were all on same line

Reply:
updated with `black`

Reply:
This is how it looks now.

![Screenshot 2021-07-11 at 12 46 58 PM](https://user-images.githubusercontent.com/6472290/125190894-2aa19300-e25d-11eb-83ae-c646101b7df2.png)